File: plugins/devkit/devkit-java-tests/testData/inspections/serializableCtor/NotAnnotatedConstructor.java
import java.io.Serializable;

public class NotAnnotatedConstructor implements Serializable {

  private static final long serialVersionUID = 1L;

  final String myString;
  final Integer myInteger;
  final Boolean myBoolean;

  public <warning descr="Non-default constructor should be annotated with @PropertyMapping">NotAnnotatedConstructor</warning>(String string, Integer integer, Boolean bool) {
    myString = string;
    myInteger = integer;
    myBoolean = bool;
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/serializableCtor/NotAnnotatedConstructor.kt
import java.io.Serializable

class <warning descr="Non-default constructor should be annotated with @PropertyMapping">NotAnnotatedConstructor</warning>(val myString: String,
                                                                                                                           val myInteger: Int,
                                                                                                                           val myBoolean: Boolean) : Serializable {

  companion object {
    private const val serialVersionUID = 1L
  }
}
